apps/src/flowlab/views/flow/nodes/ResultNode.module.scss

```diff
@@ -0,0 +1,3 @@
+.copytoClipboardButton span {
+  font-size: 0.5rem !important;
```

Comment:
Is the `!important` for overriding DSCO styles?

Reply:
Good question.  It was necessary, but obviously not ideal, so I found another way to give it precedence.